src/components/Apps/AppNavButtons.jsx

```diff
@@ -48,7 +48,10 @@ class AppNavButton extends Component {
 
     return (
       <div className={`coz-nav-apps-btns${isHomeApp ? ' --currentHome' : ''}`}>
-        <a href={homeApp && homeApp.href} className="coz-nav-apps-btns-home">
+        <a
```

Comment:
Even with an empty `href` attribute, this `<a />` will still be clickable.

I suggest instead:
```jsx
const homeHref = !isPublic && homeApp && homeApp.href
// ...
{ homeHref ?
   <div href={homeHref} className="coz-nav-apps-btns-home">
      <HomeIcon currentColor />
   </div>
   : <span className="cozy-nav-apps-btns-home">
       <HomeIcon currentColor />
     </span>
}
```
We repeat a little code but I think using a more advanced pattern does not worth it here.

Reply:
Just remove the `href` here is enough since you can detect the miss of the attribute for css and for the browser an `<a>` without href is "kind like" just a text at the end